components/site/src/lib.rs

```diff
@@ -528,11 +528,16 @@ impl Site {
         self.copy_static_directories()
     }
 
-    pub fn compile_sass(&self, base_path: &PathBuf) -> Result<()> {
+    pub fn compile_sass(&self, base_path: &Path) -> Result<()> {
         ensure_directory_exists(&self.output_path)?;
 
-        let base_path = base_path.to_string_lossy().replace("\\", "/");
-        let sass_glob = format!("{}/{}", base_path, "sass/**/*.scss");
+        let sass_path = {
+            let mut sass_path = base_path.to_owned();
```

Comment:
`to_owned()` -> `to_string()` just to be consistent with the rest of the code

Reply:
This shouldn't be a `String` -- `to_owned` here returns a `PathBuf`.

To make it more clear, I've tried changing this to `PathBuf::from(...)`. What do you think?

Reply:
woops sorry